### ISSUE TITLE
Zoom Content

### ISSUE THREAD

**Issue by ruahuy** [calvinaquino/LNReader-Android#271]
When I try to zoom the content, the text dont size up and reflow like in the older version. Is that function removed ?

**Comment by Nandaka** [comment]
Are you using Android 7+? If yes, looks like it changed the way the browser report the viewport size causing the javascript failed to reflow.

I still not sure how to change it.

**Comment by ruahuy** [comment]
Yes my android ver is 7.0 and my screen is only 5" so the text size is really hurting my eyes @@
I hope you find a way to fix it soon.

**Comment by Nandaka** [comment]
Have you tried to reset the default zoom? It will recalculate based on screen density (backup you DB first).

Else you can try to zoom in and go back to the previous screen (main menu/prev chapter) and then go foward again. It will reflow with the new zoom.

**Comment by ruahuy** [comment]
Zooming in, going back and forward somehows works. But when I overdo it and want to zoom out, it doesn't work. Neither the above method or reset to default zoom work.
